From 721886265be1fa505f55a7797e089d426459339c Mon Sep 17 00:00:00 2001
From: Richard Purdie <richard.purdie@linuxfoundation.org>
Date: Fri, 29 Jun 2018 08:02:16 +0000
Subject: [PATCH] xtrans: Fix multilib .pc file conflict

The xtrans.pc file is installed into a non-arch directory yet contains libdir
which can vary depending on which multilib is configured. The .pc file does
not require libdir so remove this to fix multilib builds.

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
Upstream-Status: Pending
---
 xtrans.pc.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/xtrans.pc.in b/xtrans.pc.in
index cf3e44f..7752996 100644
--- a/xtrans.pc.in
+++ b/xtrans.pc.in
@@ -1,6 +1,5 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
 includedir=@includedir@
 
 Name: XTrans
